metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/EnrichmentWorkerImpl.java

```diff
@@ -69,86 +55,103 @@
   }
 
   @Override
-  public byte[] process(InputStream inputStream) throws DereferenceOrEnrichException {
-    return process(inputStream, Mode.DEREFERENCE_AND_ENRICHMENT);
+  public byte[] process(InputStream inputStream)
+      throws EnrichmentException, DereferenceException, SerializationException {
+    return process(inputStream, new HashSet<>(Arrays.asList(Mode.values())));
```

Comment:
Perhaps this is cleaner:
EnumSet.allOf(Mode.class);

Reply:
Indeed, that is cleaner, will change it